src/artrace.py

```diff
@@ -2354,3 +2355,37 @@ def phys_to_pix(array, pixlocn, axis):
     else:
         pixarr = arcytrace.phys_to_pix(array, diff)
     return pixarr
+
+def slit_image(slf, det, scitrace, obj, tilts=None):
```

Comment:
yes, I believe it assumes the slit is a straight line in the calculation of each pixel.  It doesn't
assume, however, that the slit tilts the same way for each pixel (which might be worse).
Adding some docs